database/gorm/query.go

```diff
@@ -1017,6 +1360,37 @@ func filterFindConditions(conds ...any) error {
 	return nil
 }
 
+func getModelConnection(model any) (string, error) {
```

Comment:
Fix:
1. Framework can't identify the connection of slice model;
2. Framework can't switch connections when the driver is the same;